src/BenchmarkDotNet/Running/BuildPartition.cs

```diff
@@ -18,12 +19,20 @@ namespace BenchmarkDotNet.Running
 {
     public class BuildPartition
     {
+        // We use an auto-increment global counter instead of Guid to guarantee uniqueness per benchmark run (Guid has a small chance to collide),
```

Comment:
Is the reason behind the two bug reports a Guid collision?

Reply:
No, the Guid collision is a side issue. The issues are the program name itself (and thus the output folder based on that program name).